integrations/api_repo_test.go

```diff
@@ -33,22 +33,124 @@ func TestAPIUserReposNotLogin(t *testing.T) {
 	}
 }
 
-type searchResponseBody struct {
-	ok   bool
-	data []api.Repository
-}
-
-func TestAPISearchRepoNotLogin(t *testing.T) {
+func TestAPISearchRepo(t *testing.T) {
 	prepareTestEnv(t)
 	const keyword = "test"
 
 	req := NewRequestf(t, "GET", "/api/v1/repos/search?q=%s", keyword)
 	resp := MakeRequest(t, req, http.StatusOK)
 
-	var body searchResponseBody
+	var body api.SearchResults
 	DecodeJSON(t, resp, &body)
-	for _, repo := range body.data {
-		assert.True(t, strings.Contains(repo.Name, keyword))
+	assert.NotEmpty(t, body.Data)
+	for _, repo := range body.Data {
+		assert.Contains(t, repo.Name, keyword)
+		assert.False(t, repo.Private)
+	}
```

Comment:
This test should also be added to your testCases list

Reply:
I am thinking (last few days) about extracting this change to new PR because actually it's fix for non working test.